<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerDialog</class>
 <widget class="QDialog" name="TriggerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="currentModeLabel">
       <property name="text">
        <string>当前模式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="currentModeComboBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>禁用</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>启用</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="UDSRadioButton">
       <property name="text">
        <string>UDS</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="otherRadioButton">
       <property name="text">
        <string>其他</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="condGroupBox">
     <property name="title">
      <string>条件</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="condFrameIDLabel">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>帧ID：0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="condFrameIDLineEdit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="condDLCLabel">
          <property name="text">
           <string>DLC：0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="condDLCLineEdit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="condFrameTypeLabel">
          <property name="text">
           <string>帧类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="condFrameTypeComboBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>标准帧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>扩展帧</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="condFrameFormatLabel">
          <property name="text">
           <string>帧格式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="condFrameFormatComboBox">
          <item>
           <property name="text">
            <string>数据帧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>远程帧</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="condDataLabel">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="condDataLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="trigGroupBox">
     <property name="title">
      <string>触发</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="trigFrameIDLabel">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>帧ID：0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="trigFrameIDLineEdit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="trigDLCLabel">
          <property name="text">
           <string>DLC：0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="trigDLCLineEdit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="trigFrameTypeLabel">
          <property name="text">
           <string>帧类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="trigFrameTypeComboBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>标准帧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>扩展帧</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="trigFrameFormatLabel">
          <property name="text">
           <string>帧格式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="trigFrameFormatComboBox">
          <item>
           <property name="text">
            <string>数据帧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>远程帧</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="trigDataLabel">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="trigDataLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="trigSendFormatLabel">
          <property name="text">
           <string>发送格式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="trigSendFormatComboBox">
          <item>
           <property name="text">
            <string>正常发送</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>单次发送</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>自发自收</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>单次自发自收</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="trigSendNumLabel">
          <property name="text">
           <string>发送次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="trigSendNumLineEdit">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="trigTimeIntervalLabel">
          <property name="text">
           <string>时间间隔(ms)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="trigTimeIntervalLineEdit">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="additionButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TriggerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TriggerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
